extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/handler/AstroThingHandler.java

```diff
@@ -338,10 +337,7 @@ public ExpressionThreadPoolExecutor getScheduler() {
      * @return {@code true} if the {@code job} is added to the queue, otherwise {@code false}
      */
     public boolean addJobToQueue(Job job) {
-        if (job != null && !scheduledJobs.contains(job)) {
-            return scheduledJobs.add(job);
-        }
-        return false;
+        return Optional.of(job).map(scheduledJobs::add).isPresent();
```

Comment:
This had nothing to do with this PR and it makes the code smaller, but unreadable. It takes a few minutes to understand whereas the previous code was straightforward to understand. Can you please revert this method? Also I am not 100% sure it does the same as before...